Frends.RabbitMQ.Publish/Frends.RabbitMQ.Publish.Tests/QuorumQueueTests.cs

```diff
@@ -0,0 +1,350 @@
+        Assert.IsTrue(readValues.Headers.ContainsKey("Content-Type") && readValues.Headers.ContainsValue("content type")
+            && result.Headers.ContainsKey("Content-Type") && result.Headers.ContainsValue("content type"));
+        Assert.IsTrue(readValues.Headers.ContainsKey("Content-Encoding") && readValues.Headers.ContainsValue("content encoding")
+            && result.Headers.ContainsKey("Content-Encoding") && result.Headers.ContainsValue("content encoding"));
+        Assert.IsTrue(readValues.Headers.ContainsKey("X-CorrelationId") && readValues.Headers.ContainsValue("correlation id")
+            && result.Headers.ContainsKey("X-CorrelationId") && result.Headers.ContainsValue("correlation id"));
+        Assert.IsTrue(readValues.Headers.ContainsKey("X-Expiration") && readValues.Headers.ContainsValue("100")
+            && result.Headers.ContainsKey("X-Expiration") && result.Headers.ContainsValue("100"));
+        Assert.IsTrue(readValues.Headers.ContainsKey("X-MessageId") && readValues.Headers.ContainsValue("message id")
+            && result.Headers.ContainsKey("X-MessageId") && result.Headers.ContainsValue("message id"));
+        Assert.IsTrue(readValues.Headers.ContainsKey("Custom-Header") && readValues.Headers.ContainsValue("custom header")
+            && result.Headers.ContainsKey("Custom-Header") && result.Headers.ContainsValue("custom header"));
+    }
+
+    [TestMethod]
+    public void TestPublishAsByteArray()
+    {
+        Connection connection = new()
+        {
+            Host = TestHost,
+            Username = "agent",
+            Password = "agent123",
+            RoutingKey = "quorum",
+            QueueName = "quorum",
+            Create = false,
+            Durable = false,
+            AutoDelete = false,
+            AuthenticationMethod = AuthenticationMethod.Host,
+            ExchangeName = ""
+        };
+
+        Input input = new()
+        {
+            DataByteArray = Encoding.UTF8.GetBytes("test message"),
+            InputType = InputType.ByteArray,
+
+            Headers = new Header[] {
+            new Header { Name = "X-AppId", Value = "application id" },
+            new Header { Name = "X-ClusterId", Value = "cluster id" },
+            new Header { Name = "Content-Type", Value = "content type" },
+            new Header { Name = "Content-Encoding", Value = "content encoding" },
+            new Header { Name = "X-CorrelationId", Value = "correlation id" },
+            new Header { Name = "X-Expiration", Value = "100" },
+            new Header { Name = "X-MessageId", Value = "message id" },
+            new Header { Name = "Custom-Header", Value = "custom header" }
+        }
+        };
+
+        var readValues = new Helper.ReadValues();
+        var result = RabbitMQ.Publish(input, connection);
+        Helper.ReadMessage(readValues, connection);
+
+        Assert.IsTrue(!string.IsNullOrEmpty(readValues.Message) && readValues.Message.Equals("test message"));

```

Comment:
Please move these checks to their own asserts and use `AreEqual` for equal comparisons.